documentation/first-steps/README.md

```diff
@@ -61,24 +61,23 @@ Note that this is not a complete JAVA code line yet. It will be extended in the
 Every query must specify a spatial extent and a timestamp (or time range) for which the OSM data should be analyzed. This can be done by calling the methods `areaOfInterest` and `timestamps` on the object we got from the View in the previous step. The spatial extent can either be defined as a rectangular bounding box, or as an arbitrary polygonal [JTS](https://projects.eclipse.org/projects/locationtech.jts) geometry object. In our example, we will use a simple bounding box. The timestamp can be defined by providing a date-time string.
 
 ```java
-    .areaOfInterest(new OSHDBBoundingBox(8.6634, 49.3965, 8.7245, 49.4268))
-    .timestamps("2019-01-01")
+    .areaOfInterest(OSHDBBoundingBox.bboxWgs84Coordinates(8.6634, 49.3965, 8.7245, 49.4268))
+    .timestamps("2021-01-01")
 ```
 
 Note that the OSHDB expects (and returns) coordinates in the cartesian XY order: longitude first, then latitude. So, the parameters for specifying a bounding box in the OSHDB are in the following order: left, bottom, right, top (or: west, south, east, north). Just like the OpenStreetMap data, the OSHDB also works directly with coordinates in WGS84 coordinates ([EPSG:4326](https://epsg.io/4326)) of longitude and latitude. To quickly get the coordinates of a bounding box in this format, we recommend the following online tool: http://norbertrenner.de/osm/bbox.html
 
-For now, we only define a single timestamp in our query. The OSHDB also supports querying the OSM history data for multiple timestamps at once. For example, one can analyze the data in yearly steps between 2012 and 2019. At the end of this tutorial we will show what (few) changes are necessary to let our query generate the results for many timestamps at once.
+For now, we only define a single timestamp in our query. The OSHDB also supports querying the OSM history data for multiple timestamps at once. For example, one can analyze the data in yearly steps between 2012 and 2021. At the end of this tutorial we will show what (few) changes are necessary to let our query generate the results for many timestamps at once.
 
 ## 6. Filtering OSM data
 
-There are several ways to select a specific subset of the OSM dataset. In our example, we only want to look at OSM way objects which have the `building` tag. To filter these objects for our query, we add the following statements to our query:
+In our example, we only want to look at OSM way objects which have the `building` tag. To filter these objects for our query, we add the following statements to our query:
 
 ```java
-    .osmType(OSMType.WAY)
-    .osmTag("building")
+    .filter("type:way and building = *")
```

Comment:
are those whitespaces around `=` intentional? The other filters don't have them.

Reply:
not really, was a typo. this is better:
```suggestion
    .filter("type:way and building=*")
```